manifests/2.18.0/opensearch-2.18.0-test.yml

```diff
@@ -6,6 +6,9 @@ ci:
     name: opensearchstaging/ci-runner:ci-runner-al2-opensearch-build-v1
     args: -e JAVA_HOME=/opt/java/openjdk-21
 components:
+  - name: opensearch
+    smoke-test:
+      test-spec: opensearch.yml
```

Comment:
Maybe move to 2.19 now?

Reply:
We haven't started the release cycle for 2.19 so there is no complete distribution bundle for testing purpose yet so I used 2.18 to demonstrate the correct command to start the workflow. I'm OK to change this to 2.19; and we should also update the manifest workflow to attach the smoke tests entry into the test manifest creation.

Reply:
Moved to 2.19 test manifest.

Reply:
Confirmed new command `./test.sh smoke-test  manifests/2.19.0/opensearch-2.19.0-test.yml --paths opensearch=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.19.0/10545/linux/x64/tar/` is running correctly.